src/main/java/mods/battlegear2/asm/MethodMapping.java

```diff
@@ -0,0 +1,32 @@
+package mods.battlegear2.asm;
+
+import mods.battlegear2.asm.loader.BattlegearLoadingPlugin;
+
+public enum MethodMapping {
+
+    // spotless:off
+    ENTITYAICONTROLLEDBYPLAYER$UPDATETASK("e()V", "updateTask()V", 1),
```

Comment:
Can you add underscores here to separate the words? This is near-impossible to read